styles.css

```diff
@@ -0,0 +1,183 @@
+.wrapper{
+  margin: auto;
+  width: 80%;
+}
+.header{
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 100px;
+}
+.logo{
+  position: relative;
+  padding: 40px 0;
+  padding-left: 44px;
+  font: bold 24px/31px 'PT Serif',serif;
```

Comment:
Codacy found an issue: [Expected single space after "," in a single-line list (value-list-comma-space-after)](https://app.codacy.com/gh/WebHeroSchool/site99/pullRequest?prid=6205931)